system/include/emscripten/val.h

```diff
@@ -351,6 +353,14 @@ class val {
         return val(internal::_emval_new_object());
     }
 
+    static val u8string(const char* s) {
+        return val(internal::_emval_new_u8string(s));
+    }
+
+    static val u16string(const char16_t* s) {
+        return val(internal::_emval_new_u16string(s));
+    }
```

Comment:
Oh, I think these should be documented here: https://emscripten.org/docs/api_reference/val.h.html (in the repo, under `site/`).

Reply:
OK.